charts/gateway/README.md

```diff
@@ -1157,6 +1159,49 @@ The full default is this
     com.l7tech.server.clusterStaleNodeCleanupTimeoutSeconds=86400
     # Additional properties go here
 ```
+### DISKLESS Configuraton
+DISKLESS_CONFIG is a flag to pass gateway sensitive data to be mounted to the container gateway file system or passed as environment variables.
```

Comment:
Should be kept disklessConfig to reduce confusion.

Reply:
DISKLESS_CONFIG is the actual environment variable being set but yes, we can discuss this in TechDocs instead.